doc/docs/master/how-tos/developer-workflow/working-with-pipelines.md

```diff
@@ -1,19 +1,18 @@
-# Individual Developer Workflow
+# Working with Pipelines
```

Comment:
I wonder if this section should be part of the index/overview? Index is pretty long already so maybe not.

Reply:
Originally this was part of the index, but then it felt a little too long to power through to get to the automation steps. I'm definitely torn on this one though, as it does make sense to introduce what working with a pipeline looks like before talking about how to automate it. Is there a way to track the order of clicks in the docs? It's probably too much effort, but I'd be curious to see how people are actually looking through things now to find out how to structure it best.